server/static/jsx/scholarship.jsx

```diff
@@ -0,0 +1,96 @@
+/** @jsx React.DOM */
+define(['ext/jquery', 'ext/react'], function($, React) {
+  var ScholarshipContainer = React.createClass({
+    render: function() {
+      var scholarships = this.props.scholarshipData.map(function(data) {
+        return <ScholarshipBox data={data} />;
+      });
+
+      return (
+        <div>
+          <h1> Scholarships </h1>
+          <div className="scholarship-container">{scholarships}</div>
+        </div>
+      );
+    }
+  });
+
+  var ScholarshipBox = React.createClass({
+    getInitialState: function() {
+      return {
+        expanded: false,
+        removed: false
+      }
+    },
+
+    toggleExpansion: function() {
+      this.setState({expanded: !this.state.expanded});
+    },
+
+    removeFromProfile: function() {
+      this.setState({removed: true});
+      $.ajax({
+        type: 'PUT',
+        url: '/api/v1/user/remove_scholarship_from_profile',
+        data: {
+          scholarship_id: this.props.data.id
+        }
+      });
+    },
+
+    render: function() {
+      if (this.state.removed) {
+        return null;
+      }
+
+      var cx = React.addons.classSet;
+      var classes = cx({
+        'scholarship-content': true,
+        'expanded': this.state.expanded
+      });
+
+      scholarshipInner = null;
+
+      if (this.state.expanded) {
+        scholarshipInner = (
+          <div className="scholarship-inner row-fluid">
+            <div className="span8 left-col">
+              {this.props.data.description.replace('&amp;', 'and')}
+            </div>
+            <div className="span4 right-col">
+              <ul>
+                {this.props.data.eligibility.concat(
+                    this.props.data.enrollment_year).map(function(req) {
+                  return (<li>{req}</li>);
+                })}
+              </ul>
+            </div>
+            <div className="row-fluid">
+              <div className="span12 more-info">
+                <a href={this.props.data.link}>
+                  <i className="icon-info-sign"></i> More Info</a>
+                <a onClick={this.removeFromProfile}>
+                  <i className="icon-remove-sign"></i> Remove from profile</a>
```

Comment:
nit: `</a>` should be on its own line to line up with opening tag. Also applies to above.